index.js

```diff
@@ -105,7 +105,7 @@ LDIterator.prototype._next = function (callback) {
 
         self.initCompleted = true;
         var i = -1;
-        while (++i < self.onInitCompleteListeners) {
+        while (++i < self.onInitCompleteListeners.length) {
```

Comment:
once we get coverage tests in (#72), this should be a lot easier to catch and measure. it was sloppy of us not to test this in the past.